extensions/azdata/src/typings/azdata-ext.d.ts

```diff
@@ -127,16 +127,24 @@ declare module 'azdata-ext' {
 			uid: string // "cea737aa-3f82-4f6a-9bed-2b51c2c33dff"
 		},
 		spec: {
-			limits?: {
-				memory?: string // "10Gi"
-				vcores?: string // "4"
-			},
-			requests?: {
-				memory?: string // "10Gi"
-				vcores?: string // "4"
+			scheduling?: {
+				default?: {
+					resources?: {
+						limits?: {
+							memory?: string // "10Gi"
+							cpu?: string // "4"
+						},
+						requests?: {
+							memory?: string // "10Gi"
+							cpu?: string // "4"
+						}
+					}
+				}
 			}
-			service: {
-				type: string // "NodePort"
+			services: {
+				primary: {
+					type: string // "NodePort"
+				}
 			}
 			storage: {
```

Comment:
There should be changes to `spec.storage.*`.  It's now `spec.storage.*.volumes[]`